src/MICore/JsonLaunchOptions.cs

```diff
@@ -137,6 +137,10 @@ public override object ReadJson(JsonReader reader, Type objectType, object exist
             {
                 visualizerFile.Add(reader.Value.ToString());
             }
+            else
+            {
+                throw new Exception("Input is neither a List nor a String");
```

Comment:
You will want to Localize this.

Add an entry in https://github.com/microsoft/MIEngine/blob/5b50551e6827381fcd238c3669142fab0bcc8097/src/MICore/MICoreResources.resx